src/integration/socks-echo/integration.ts

```diff
@@ -5,6 +5,8 @@
 /// <reference path="../../tcp/tcp.d.ts" />
 /// <reference path="../../webrtc/peerconnection.d.ts" />
 
+freedom['loggingprovider']().setConsoleFilter(['*:D']);
```

Comment:
Definitely keep debug for these tests. Thanks for adding, this is useful for others to quickly change the logging level, too.